roles/network_plugin/calico/defaults/main.yml

```diff
@@ -12,6 +12,10 @@ ipip_mode: "{{ 'Always' if ipip else 'Never' }}"  # change to "CrossSubnet" if y
 calico_ipip_mode: "{{ ipip_mode }}"
 calico_vxlan_mode: 'Never'
 
+calico_ipip_mode_ipv6: Never
```

Comment:
may be set 
calico_ipip_mode_ipv6: "{{ ipip_mode }}" 

to enable tunneling for ipv6?

Reply:
It's currently not supported from what I can tell: https://github.com/projectcalico/libcalico-go/issues/996
It certainly produced an error when I was using `ipip_mode` originally